### DISCUSSION TITLE
Ta Serto #1

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#390] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/ta-serto/

**Comment by angellisy** [comment]
Tadinho do Luís :(

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/ta-serto/#comment_1087753953>):

> Tadinho do Luís :(

Calma, não é desse Luís que eu tava falando



[![null](<https://camo.githubusercontent.com/b03f791a9c9b343f69c8ea980474ef76b108363825c0a4070ebd5b897624ff65/68747470733a2f2f692e706f7374696d672e63632f6a53363157624b472f696d2d776974682d796f752e706e67>)](<https://camo.githubusercontent.com/b03f791a9c9b343f69c8ea980474ef76b108363825c0a4070ebd5b897624ff65/68747470733a2f2f692e706f7374696d672e63632f6a53363157624b472f696d2d776974682d796f752e706e67>)

**Comment by tali-zorah** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/ta-serto/#comment_1087760722>):

> Calma, não é desse Luís que eu tava falando[![null](<https://camo.githubusercontent.com/b03f791a9c9b343f69c8ea980474ef76b108363825c0a4070ebd5b897624ff65/68747470733a2f2f692e706f7374696d672e63632f6a53363157624b472f696d2d776974682d796f752e706e67>)](<https://camo.githubusercontent.com/b03f791a9c9b343f69c8ea980474ef76b108363825c0a4070ebd5b897624ff65/68747470733a2f2f692e706f7374696d672e63632f6a53363157624b472f696d2d776974682d796f752e706e67>)

*Keelah!* De qual Luís você dois estão falando?

**Comment by brunoaugusto** [comment]
> [Tali'Zorah nar Rayya](</perfil/?user=tali-zorah>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/ta-serto/#comment_1087762203>):

> *Keelah!* De qual Luís você dois estão falando?

Oras, do único Luís que importa



![null](<https://i.postimg.cc/bv3v8W5h/antman-luis.png>)
